docs/advanced-install-options.md

```diff
@@ -73,62 +73,9 @@ To use the bundle:
-
-Before proceeding, you should ensure that [.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) or later is installed.
-
-As an optional step, if you want to invoke a Python script for non-default monitors, specifically **exec**, then you must have Python installed.
-
-To install the MSI package (requires user with Administrator privileges):
-
-1. Double-click the downloaded MSI package to install the agent. The files will be installed to `\Program Files\SignalFx\SignalFxAgent`.
-
-2. Ensure a valid configuration file is available somewhere on the target system. By default, the Smart Agent will look for a configuration file at `\ProgramData\SignalFxAgent\agent.yaml`. A sample configuration file can be found at `\Program Files\SignalFx\SignalFxAgent\etc\signalfx\agent.yaml`. See [Configuration](#configuration) for more information on how to configure the Smart Agent.
-
-3. Run the Smart Agent by invoking the Smart Agent executable:
-
-```sh
-PS> \Program Files\SignalFx\SignalFxAgent\bin\signalfx-agent.exe -config <path to config file>
-
-```
-By default, the Smart Agent runs as a process in the foreground and logs only to stdout/err.
-
-You may optionally install the Smart Agent as a Windows service and log the output to the Windows Event Viewer by invoking the agent executable and specifying a few command line flags. The examples below show how to install and start the Smart Agent as a Windows service.
-
-_Install Service_
-
-```sh
-PS> \Program Files\SignalFx\SignalFxAgent\bin\signalfx-agent.exe -service "install" -logEvents -config <path to config file>
-````
-
-_Start Service_
-
-```sh
-PS> \Program Files\SignalFx\SignalFxAgent\bin\signalfx-agent.exe -service "start"
-````
-
-To stop or uninstall the Smart Agent service:
-
-_Stop Service_
-
-```sh
-PS> \Program Files\SignalFx\SignalFxAgent\bin\signalfx-agent.exe -service "stop"
-````
-
-_Uninstall Service_
-
-```sh
-PS> \Program Files\SignalFx\SignalFxAgent\bin\signalfx-agent.exe -service "uninstall"
-````
-
-4. The `SignalFx Smart Agent` package can be uninstalled from "Programs and Features" in the Windows Control Panel (if the Smart Agent service is installed, it will be automatically stopped and uninstalled).
-
-
-### Windows Standalone .zip (for Smart Agent TBD or older)
-
-If you don’t want to use the installer script, we offer a .zip that can be deployed to the target host. This bundle is available for download on the [Github Releases Page](https://github.com/signalfx/signalfx-agent/releases) for versions TBD or older.
+If you are unable to use the installer script or Chocolatey, we offer a .zip that can be deployed to the target host. This bundle is available for download on the [Github Releases Page](https://github.com/signalfx/signalfx-agent/releases).
```

Comment:
See my previous note. This sentence implies that you should only use the `.zip` file if the other two alternatives don't work. Is that true? If not, drop the opening phrase.